clinvar_ingest/main.py

```diff
@@ -85,6 +85,7 @@ def run_parse(args):
     """
     Primary entrypoint function. Takes CLI arg vector excluding program name.
     """
+    # if we're running this as a cloud function, it probably makes sense to remove all local filesytem ops?
```

Comment:
Yes we'll probably want a way to swap between "file://" and other file protocols like "gs://"

This is how we did it in the `clinvar-streams` project and it worked well. With a simple config value on the inputs you could make it work fully locally or fully using buckets.

https://github.com/clingen-data-model/clinvar-streams/blob/3bd302a81f70221bc7a5f800dec2572af9f36f3a/src/clinvar_raw/stream.clj#L127-L136